testdata/config/quota.yaml

```diff
@@ -25,7 +25,7 @@ metadata:
   name: quota
   namespace: istio-config-default
 spec:
-#  selector: request.headers["user"] != "admin"
+#  match: source.user  != "admin"
```

Comment:
Same as above

Reply:
I think it is useful to have an example in the checked in file. 
This says impose quota if not admin user.